addons/web_editor/views/snippets.xml

```diff
@@ -380,7 +380,7 @@
 
     <!-- Replace a media -->
     <!-- TODO probably review this system once the new editor is merged to not duplicate the selector, etc -->
-    <div data-js="ReplaceMedia" data-selector="img, .media_iframe_video, span.fa, i.fa">
+    <div data-js="ReplaceMedia" data-selector="img, .media_iframe_video, span.fa, i.fa" data-exclude="a[href^='/website/social/'] > i.fa">
```

Comment:
Maybe there is better to do functionally here:
- s_share snippet -> if we disable for the social media, why not for the s_share snippet?
- we can still edit the icon via double click, maybe the replace button should also be kept and just disabling the link edition part of the ReplaceMedia option ? Actually the replace button makes even more sense for custom social media...

Reply:
- AFAIK the only way for the user to change the url of a link of share snippet is to do it with the media link tool.
- Actually the replace media button and the media link tool are still available except for the social networks which are in DB. Indeed it is not consistent that you can double click and still replace the icon (I totally agree with you). Apart from that, the behavior seems consistent, the url stored in DB in social_facebook will always have the facebook icon.

Reply:
> AFAIK the only way for the user to change the url of a link of share snippet is to do it with the media link tool.

Well that's the point, you don't want the user to be able to do that anymore since there is your new snippet for that. The s_share... is to share, we don't want to allow editing the links (nor do we want the user to be able to see strange technical sharing links).

> Actually the replace media button and the media link tool are still available except for the social networks which are in DB.

Mmh indeed, missed that somehow.

> Indeed it is not consistent that you can double click and still replace the icon (I totally agree with you). Apart from that, the behavior seems consistent, the url stored in DB in social_facebook will always have the facebook icon.

I guess it's ok, let's say the double click is a hidden feature :+1: (Note that there are multiple facebook icons or you may want to change for a custom one).

Reply:
> Well that's the point, you don't want the user to be able to do that anymore since there is your new snippet for that. The s_share... is to share, we don't want to allow editing the links (nor do we want the user to be able to see strange technical sharing links).

Well, I'm not sure I follow you... :grimacing:  If we disable link modification for share snippet, how is the user supposed to set this block with his own social networks? (I think if  we do that the share block will not be usable at all) 
We could propose a solution similar to this new social_media block.(Have a we-list that allows to manage the different share_link easily etc).  I think this is beyond the scope of the task no?

OK for the rest :+1:

Reply:
I don't understand what you don't understand :shrug: The share snippet has nothing to do with "your own social networks", we have your snippet for that. Did you try to use the s_share snippet before sending this message?

You add it in the page you have
![image](https://user-images.githubusercontent.com/10338094/150364294-fa39240e-e1ee-4cc9-8570-c21a77690534.png)

Click on a share link you have this:
![image](https://user-images.githubusercontent.com/10338094/150364932-6cede783-2541-42fa-a8e2-086f7ef3b48b.png)

=> You can remove the share link you don't want, there is a trash icon (but this is not linked to your own social media, normally there is no real point in removing some sharer, you can let people share your super website on their own social media even if you do not have an account over there yourself).

=> You see a strange URL that the user cannot understand... if he changes it, the share is broken -> why would you allow it? It was necessary before because the share snippet was our way to display your own social media and thus you could edit those links... but we have your new snippet now

Reply:
Ok MB, I actually got confused between social networks and sharing on a social network. I understand and agree your point now :+1:

Reply:
Just a small question about this: As there is a mailto in the share block... Is it better to disable the media link tool for share link that are not mailto or wouldn't it be better to remove this mailto from the share block ?

Reply:
Unless I am missing something, you are still confused between social networks and sharing on a social network... why would you change that share `:mailto` which is... to share the current page by mail?